generators/server/templates/_build.gradle

```diff
@@ -208,6 +208,10 @@ dependencies {
     <%_ if (authenticationType == 'jwt') { _%>
     compile group: 'io.jsonwebtoken', name: 'jjwt', version: jjwt_version
     <%_ } _%>
+    <%_ if (applicationType == 'uaa' || authenticationType == 'uaa') { _%>
```

Comment:
an uaa application has always authenticationType uaa, but also other applications can have this type.

it's not always true, but authenticationType === 'uaa' is indeed enough